### PR TITLE
chore: story feed tracking

### PR DESCRIPTION
**Pull request by sruddy**

### CODE REVIEW

File: packages/dapp/src/components/StoryFeed/StoryDetails.tsx [hunk 1]
```diff
@@ -53,7 +57,13 @@ export const StoryDetails: React.FunctionComponent<StoryDetailsProps> = props =>
       </StoryDetailsFullBleedHeader>
       <StoryDetailsContent>
         <StoryDetailsFlex>
-          <StoryLink href={openGraphData.url} target="_blank">
+          <StoryLink
+            onClick={() =>
```
Comment (dankins) [inline packages/dapp/src/components/StoryFeed/StoryDetails.tsx]: can you confirm that this doesn't cause any issues having an href and an onclick handler? Just want to make sure this doesn't stop event propogation or break the link, for example

Comment (sruddy) [reply packages/dapp/src/components/StoryFeed/StoryDetails.tsx]: I tested it, I see the GA response and the link is working.

**Comment by dankins** [comment]
:tada: This PR is included in version 4.9.8 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**